introspection.go

```diff
@@ -266,6 +266,14 @@ mutation operations.`,
 					return nil, nil
 				},
 			},
+			"subscriptionType": &Field{
+				Description: `If this server support subscription, the type that ' +
+                   'subscription operations will be rooted at.`,
+				Type: __Type,
+				Resolve: func(p ResolveParams) (interface{}, error) {
+					return nil, nil
+				},
+			},
```

Comment:
Alright @pyros2097 thanks for insights, I think we can keep this one as well, since there's on going graphiql support (https://github.com/graphql/graphiql/issues/55 & https://github.com/graphql/graphiql/issues/79) for graphql servers which does not support subscription feature yet.